src/components/UpdatesFeed/UpdateCard/index.js

```diff
@@ -0,0 +1,173 @@
+import PropTypes from "prop-types";
+import { useEffect, useRef, useState } from "react";
+import { makeStyles } from "tss-react/mui";
+
+import Button from "@mui/material/Button";
+
+import { getProjectIcon } from "../services";
+import ReadMoreButton from "./ReadMoreButton";
+
+const useStyles = makeStyles()((theme) => ({
+	container: {
+		color: "black",
+		background: "white",
+		padding: "0.75rem 0.75rem 0 0.75rem",
+		display: "flex",
+		gap: "0.625rem",
+		flexDirection: "row",
+		"& svg": {
+			color: "#585858",
+			fontSize: "2rem",
+		},
+	},
+	details: {
+		display: "flex",
+		flexDirection: "column",
+		width: "100%",
+	},
+	detailsHeader: {
+		alignItems: "center",
+		display: "flex",
+		color: theme.palette.text.primary,
+		width: "100%",
+		justifyContent: "space-between",
+	},
+	projectName: {
+		fontWeight: 700,
+		fontSize: "1.375rem",
+		color: "#195A96",
+	},
+	subheader: {
+		display: "flex",
+		gap: "1rem",
+		fontWeight: 700,
+		justifyContent: "space-between",
+	},
+	expandedContent: {
+		overflow: "unset",
+		display: "unset",
+	},
+	updateContent: {
+		overflow: "hidden",
+		textOverflow: "ellipsis",
+		display: "-webkit-box",
+		WebkitLineClamp: "2",
+		WebkitBoxOrient: "vertical",
+	},
+	footer: {
+		display: "flex",
+		flexWrap: "wrap-reverse",
+		paddingBottom: "1rem",
+		paddingTop: "1rem",
+		justifyContent: "space-between",
+	},
+	buttonBar: {
+		display: "flex",
+		gap: "0.5rem",
+	},
+	buttonPrimary: {
+		padding: "0.75rem 1rem",
+		fontWeight: 700,
+		fontSize: "0.875rem",
+		height: "2rem",
+	},
+	buttonSecondary: {
+		padding: "0.75rem 1rem",
+		fontWeight: 700,
+		fontSize: "0.875rem",
+		height: "2rem",
+	},
+}));
+
+const UpdateCard = ({ documentUrl, pcp, project, updateContent, updateDate, updateName }) => {
+	const { classes } = useStyles();
+
+	const [isClamped, setClamped] = useState(false);
+	const [isExpanded, setExpanded] = useState(false);
+
+	const updateContentRef = useRef(null);
+
+	const formattedDate = new Intl.DateTimeFormat("en-US", {
```

Comment:
Please create `src/services/date.js` and add this as an exportable function `getLongDate(date)` or something with a better name idk.

Reply:
Thank you! I am going to flesh out the date service and add a test file. We need test files for all of our js files, but I want to add a couple first as examples before making them required in PRs.